src/docs/ftw-introduction/ftw-product/index.md

```diff
@@ -0,0 +1,132 @@
+code-chunk. That means that everything in that file is going to slow
+down page rendering - when the page is fully loaded. That also affects
+search engine optimization (SEO).
+
+With FTW-product, we have decided to move certain page-specific files
+under page-directories. Here are some of the relevant changes in the
+directory structure:
+
+- **src/config/**: Generic configuration files are moved to this
+  directory
+  - Extra configs:
+    - **mainSearchType** in Topbar. It's either _'keywords'_ or
+      _'location'_.
+    - **searchPageVariant**: _'list'_ & _'map'_.
+    - **listingPageVariant**: _'product'_ & 'TODO: hero'
+  - Renamed configs:
+    - **transactionProcessAlias** (in other templates:
+      bookingProcessAlias)
+    - **lineItemUnitType** (in other templates: bookingUnitType)
+- **src/routing**: Route configuration and UI components related to
+  routing are moved to this directory
+- UI components that are only used in a single page or within a more
+  complex component are nested within that page or component. For
+  example:
+  - _EditListinWizard_ is moved under _EditListinPage_
+  - Search map and filters are moved under _SearchPage_
+  - _TransactionPage_ got subcomponents like _ActivityFeed_,
+    _ReviewModal_, _ReviewForm_, _SendMessageForm_, and
+    _TransactionPanel_.
+- **src/forms**: this directory was removed as there was only one form
+  that was truly shared between pages: _StripeConnectAccountForm_. It's
+  moved to _src/components/_.
+
+TODO: add other notable changes here
+
+## Differences in transaction process
+
+FTW-product has different transaction process graph than what FTW-daily
+and FTW-hourly use. Visually, the biggest changes are after checkout
+page and before review period. E.g. FTW-product has dispute state there.
+However, the defining difference is actually in actions: FTW-product
+doesn't use bookings (and availability management of time-slots).
+Instead, it has actions to deal with stock management.
+
+Roughly, the transaction process graph looks like this:
+
+![flex-product-default-process](./flex-product-default-process.png)
+
+## Managing stock
+
+Stock management is pretty straightforward. On EditListingWizard, (which
+is a component to create and update listings), provider can set the
+stock quantity for the listing. Under the hood, FTW-product calls API
+with _`sdk.stockAdjustments/compareAndSet()`_ with the value provider is
+trying to set (`newTotal`) and also the previous value (`oldTotal`). At
+the time of update, if the assumed old total still matches with the
+current stock, the **currentStock** entity is updated to the value
+passed through with `newTotal` parameter. This check is done to prevent
+race conditions, where someone orders a listing while provider is
+updating its stock.
+
+Listing entity has a relationship to "currentStock" entity - so,
+currentStock info can be included listing queries. Then on ListingPage,
```

Comment:
Does it need to be explicitly included anymore?

Reply:
Yes. E.g. ListingPage.duck.js
`include: ['author', 'author.profileImage', 'images', 'currentStock'],`